modules/swagger-codegen/src/main/resources/LoopBackSwift/String+Mappable.mustache

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import ObjectMapper
+
+extension String : Mappable {
```

Comment:
@jacobrelkin Some requests response data is just a `String` or a `Bool`. To be able to return `Observable<String>` or `Observable<Bool>` from the `request` func, the data type needs to be `Mappable`.